VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
 	src/VersionControlUIException.cpp
 	src/VersionControlUIPlugin.cpp
+	src/DiffManager.cpp
 	test/SimpleTest.cpp
 )
-envision_plugin(VersionControlUI InteractionBase)
+envision_plugin(VersionControlUI InteractionBase oomodel)
```

Comment:
In DiffManager the method firstAncestorOfType<OOModel::Class> is used at the moment. To test visualization of the differences until there is maybe a more generic way to decide which context to show.